src/Microsoft.ML.StandardLearners/FactorizationMachine/FieldAwareFactorizationMachineModelParameters.cs

```diff
@@ -42,10 +42,30 @@ private static VersionInfo GetVersionInfo()
                 verReadableCur: 0x00010001,
                 verWeCanReadBack: 0x00010001,
                 loaderSignature: LoaderSignature,
-                loaderAssemblyName: typeof(FieldAwareFactorizationMachinePredictor).Assembly.FullName);
+                loaderAssemblyName: typeof(FieldAwareFactorizationMachineModelParameters).Assembly.FullName);
         }
 
-        internal FieldAwareFactorizationMachinePredictor(IHostEnvironment env, bool norm, int fieldCount, int featureCount, int latentDim,
+        public FieldAwareFactorizationMachineModelParameters(IHostEnvironment env, bool norm, int fieldCount, int featureCount, int latentDim,
```

Comment:
Doc string please. In the f-th field, the j-th feature's latent vector, `v_{j, f}` in the [doc](https://github.com/wschin/fast-ffm/blob/master/fast-ffm.pdf), starts at `latentWeights[j * fieldCount * latentDim + f * latentDim]`. The k-th element in `v_{j, f}` is `latentWeights[j * fieldCount * latentDim + f * latentDim + k]`.